### PR TITLE
Update ViewMode.java

### PR DESCRIPTION
**Pull request by mst3k4ever**
Comment edits.

### CODE REVIEW

**Review by TingChen2020 — CHANGES_REQUESTED** [review]
Some questions need further discussion before resolving. Could you please check my replies tagged with "// TODO CT"?

**Comment by mst3k4ever** [comment]
Hi. I'm not entirely certain how to edit this (there are three commits -- which do I edit, or is there still one main document?

> Some questions need further discussion before resolving. Could you please check my replies tagged with "// TODO CT"?

File: Android--EN/domain/ViewMode.java [hunk 1]
```diff
@@ -22,21 +22,30 @@
      * In this mode, the user's view follows the view of the host.
      *
      * @note
-     * - When a user’s view mode is set as `Broadcaster`, the view mode of other users in the room (including newly-joined users) automatically changes to `Follower`.
-     * - When a user in the `Follower` view mode operates the whiteboard, their view mode automatically swiches to the `Freedom` mode.
+     * - When one user’s view mode is set as `Broadcaster`, the view mode of the other users in the room (including users that join subsequently) automatically changes to `Follower`. 
+     //TODO William I don't think you need quote marks around the view modes unless you're talking about the value?
+     // TODO CT Words in quote marks will be rendered as code format in generated documents. I think `Freedom`, `Follower` and `Broadcaster` should be in code format, because they are the values of the view mode.
```
Comment (mst3k4ever) [inline Android--EN/domain/ViewMode.java:27]: I think "There can be only one user in Broadcaster view mode." is different from "The user's view mode is set to 'Broadcaster'." One is "programmatic", while the other is just a description. However, if you feel like all of this is speaking "programmatically", then it's probably okay.

Change line 25 to: "When a user’s view mode is set as 'Broadcaster', the view mode of every other user in the room (including users that join subsequently) is automatically set as 'Follower'."

File: Android--EN/domain/ViewMode.java [hunk 2]
```diff
@@ -22,21 +22,30 @@
...
     /**
-     * Host mode.
+     * Host mode. 
+     //TODO William "Broadcaster mode"?
+     // TODO CT I agree that for the consistency between the code and the mode desvription, "Broadcaster mode" is better. 
+     However, we were askded to change all "broadcaster" to "host" for the `broadcaster` user role in RTC API documentation. (the code is still `broadcaster`) I think the change is 
      *
-     * In this mode, the user can freely adjust their view and synchronize thier view to other users in the room.
-     * 该模式下用户可以主动调整视角，并将自己的视角同步给房间内所有其他用户。
+     * In this mode, the user can freely adjust their view and synchronize thier view to other users in the room.
```
Comment (mst3k4ever) [inline Android--EN/domain/ViewMode.java:41]: Typo: the second "thier" should be "their"  :-/  However, I'm rewriting that sentence.  :-)

With regards to "synchronization", you synchronize to a source (when there is one), which would be the Broadcaster/Host's view. However, I don't think it's quite right. I think "duplicate" is more accurate.

How about this (rewrite of lines 41-48):

* In this mode, the user can freely adjust their view. However, the view of every other user in the room will be restricted to duplicating the view of the host.
*
* When a user’s view mode is set as `Broadcaster`, the view mode of every other user in the room (including users that join subsequently) is automatically set as `Follower`.
*
* Each room can have only one user in `Broadcaster` view mode. 

After this, you can add in whatever new information you learn.

File: Android--EN/domain/ViewMode.java [hunk 3]
```diff
@@ -22,21 +22,30 @@
      * In this mode, the user's view follows the view of the host.
```
Comment (mst3k4ever) [inline Android--EN/domain/ViewMode.java:22]: Change "follows" to "duplicates", I think.

File: Android--EN/domain/ViewMode.java [hunk 4]
```diff
@@ -22,21 +22,30 @@
...
+     // TODO CT I think that will happen the next time when a user swithces to `Broadcaster` view mode. I will double check with the developer.
      * If needed, you can call {@link disableOperations(boolean) disableOperations} to disable the user from operating the whiteboard, so as to lock their view mode.
      */
     @SerializedName("follower")
     Follower,
     /**
-     * Host mode.
+     * Host mode. 
+     //TODO William "Broadcaster mode"?
+     // TODO CT I agree that for the consistency between the code and the mode desvription, "Broadcaster mode" is better.
```
Comment (mst3k4ever) [inline Android--EN/domain/ViewMode.java:38]: Based on your description of what they want in the docs, "Host mode" is probably better.

**Review by mst3k4ever — COMMENTED** [review]
Comments are in-line.

**Pull request closed without merging**